packages/main/test/samples/Calendar.sample.html

```diff
@@ -0,0 +1,105 @@
+<header>
+	<h2 class="control-header">Calendar</h2>
+</header>
+
+<div class="component-package">@ui5/webcomponents</div>
+
+<div class="control-tag">&lt;ui5-calendar&gt;</div>
+
+<section>
+	<h3>Basic Calendar</h3>
+	<div class="snippet">
+		<div class="datepicker-width">
+			<ui5-calendar></ui5-calendar>
+		</div>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+		<ui5-calendar></ui5-calendar>
```

Comment:
remove all the tabs in front of the ui5-calendar in the "pre" tags, otherwise it is displayed like this:


<img width="530" alt="Screenshot 2020-11-26 at 5 44 23 PM" src="https://user-images.githubusercontent.com/15702139/100370713-5b56f400-300f-11eb-8172-d8e27d17165f.png">

Reply:
Done